### PR TITLE
Add Oracle module maintainers

### PR DESCRIPTION
**Pull request by nalsaber**
##### SUMMARY

Add Oracle module maintainers in BOTMETA.yml

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
BOTMETA.yml

```

### CODE REVIEW

**Comment by nalsaber** [comment]
@gundalow can you please take a look at this?

File: .github/BOTMETA.yml [hunk 1]
```diff
@@ -1116,6 +1118,12 @@ files:
   $plugins/doc_fragments/xenserver.py:
     maintainers: bvitnik
   $plugins/doc_fragments/hpe3par.py: *hpe3par
+  $plugins/doc_fragments/oracle.py: $team_oracle
```
Comment (samdoran) [inline .github/BOTMETA.yml]: You can use a glob here. Also, please mark this as community supported:
```suggestion
  $plugins/doc_fragments/oracle*: $team_oracle
    maintainers: $team_oracle
    support: community
```

File: .github/BOTMETA.yml [hunk 2]
```diff
@@ -805,6 +806,7 @@ files:
     maintainers: $team_openstack
     labels:
       - cloud
+  $module_utils/oracle/: $team_oracle
```
Comment (samdoran) [inline .github/BOTMETA.yml]:
```suggestion
  $module_utils/oracle/:
    maintainers: $team_oracle
    support: community
```

**Review by gundalow — APPROVED** [review]

**Comment by gundalow** [comment]
@nalsaber Thanks, we've updated the format a bit

**Pull request merged**